src/Tests/Peachpie.Test/Peachpie.Test.csproj

```diff
@@ -11,6 +11,7 @@
     <ProjectReference Include="..\..\Peachpie.Library.Graphics\Peachpie.Library.Graphics.csproj" />
     <ProjectReference Include="..\..\Peachpie.Library.Network\Peachpie.Library.Network.csproj" />
     <ProjectReference Include="..\..\Peachpie.Library.Scripting\Peachpie.Library.Scripting.csproj" />
+    <ProjectReference Include="..\..\Peachpie.Library.XmlDom\Peachpie.Library.XmlDom.csproj" />
```

Comment:
not relevant to this PR but we can keep it - just be careful next time when committing something